docs/guide/SecurityBC.md

```diff
@@ -273,3 +273,234 @@ Here is the response that comes back:
+
+The Oathkeeper helm chart deploys two key components into Kubernetes: Oathkeeper itself, and the Oathkeeper Maester. Oathkeeper is stateless and config-driven, and seamlessly reloads itself with zero downtime anytime that config changes. Oathkeeper Maester is a controller for the Rule CRD, and composes the Rule objects in Kubernetes into a single complete rules file that is loaded by Oathkeeper.  By default, that’s a ConfigMap that Oathkeeper mounts, but it can also be set to run as a sidecar and use a shared mount.
+
+Oathkeeper exposes two ports as two services. One service is the API service, and the other is the Proxy service. Long-term, we’ll be using the API service, which will be queried by the next generation API Gateway using the Access Control Decision API  (see [REST API | ORY Oathkeeper](https://www.ory.sh/oathkeeper/docs/reference/api/#access-control-decision-api) ) that Oathkeeper provides, but for now we will be using the Proxy service, and exposing that via Ingress. External URLs to services protected by Oathkeeper will be pointed at  the Oathkeeper Proxy ingress, which will then proxy access to the internal services at those URLs and apply access control rules.
+
+Oathkeeper will be configured to generate and sign a JWT containing claims that internal services can check and verify by pointing at the JWKS that Oathkeeper will publish (which is part of the configuration) at the well-known URL for JWKS on the API service (see [REST API | ORY Oathkeeper](https://www.ory.sh/oathkeeper/docs/reference/api/#lists-cryptographic-keys) ). If a service does this, then it is operating under a basic zero trust regime, as it will not be possible to call that service except with a token that has been generated by Oathkeeper, and Oathkeeper will only generate a token if the access rules for the given URL allow access.
+
+
+### Debugging
+The responses and logs from Oathkeeper tend to be pretty informative, definitely start there. Make sure you’re looking at the logs for the request that matters. Oathkeeper will also be logging health checks and similar. 
+
+Some possible debugging actions that I’ve found useful in different circumstances:
+
+* make a match much, much more permissive (replacing the entire beginning with `<.*>` and then having the minimum currently unique suffix is good)
+* double check that each internal URL is the proper internal URL by checking it is accessible inside the cluster with curl
+* look at the IdP logs
+* make sure the domains and ports for the introspection and the external token endpoint are identical. Keycloak at least doesn’t like it if they are not
+* point the Oathkeeper rule at https://httpbin.org/ , usually the `/anything` path prefix which will reflect back everything it gets, making it easy to see what the service will see

```

Comment:
```suggestion
* Point the Oathkeeper rule at https://httpbin.org/, usually the `/anything` path prefix which will reflect back everything it gets, making it easy to see what the service will see.

```
I feel that something is off with this sentence. Shall we remove the "which" from "usually the `/anything` path prefix which will reflect back"?